driver/registry_default_recovery.go

```diff
@@ -41,3 +41,21 @@ func (m *RegistryDefault) AllRecoveryStrategies() (recoveryStrategies recovery.S
 	}
 	return
 }
+
+func (m *RegistryDefault) RecoveryExecutor() *recovery.HookExecutor {
+	if m.selfserviceRecoveryExecutor == nil {
+		m.selfserviceRecoveryExecutor = recovery.NewHookExecutor(m)
+	}
+	return m.selfserviceRecoveryExecutor
+}
+
+func (m *RegistryDefault) PostRecoveryHooks(ctx context.Context) (b []recovery.PostHookExecutor) {
+
+	for _, v := range m.getHooks("global", m.Config(ctx).SelfServiceFlowRecoveryAfterHooks("global")) {
```

Comment:
Replace global with https://github.com/ory/kratos/pull/1240#discussion_r619988905

Reply:
This is only in code to differentiate between strategy and "global" hooks. In the config the "hook" property can be configured directly within the "after" property.